crates/revmc-worker/src/worker/sleddb.rs

```diff
@@ -6,6 +6,7 @@ use std::sync::Arc;
 
 use super::path::sleddb_path;
 
+/// In-Memory Database for local storage of compiled multiple ExternalFns
```

Comment:
is in-memory database right expression? how about embedded database?

Reply:
You are right, changed it to embedded